scripts/tank_cmd.py

```diff
@@ -134,6 +148,18 @@ def show_help(log):
 ----------------------------------------------
 - To show this help, add a -h or --help flag.
 - To display verbose debug, add a --debug flag.
+- To provide a username and a password on the command line for authentication,
```

Comment:
I thought we decided not exposing the `--login` and `--password` options, just to encourage people to not store their password in scripts etc but rather use api scripts for that sort of use case?